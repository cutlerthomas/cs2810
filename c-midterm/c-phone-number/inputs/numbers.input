(435) 555-1698
:435:555-1698
(435) 555-16982
 801-123-9876
 801--123--9876
 801-1239-876
8-0-1-1-2-3-9-8-7-6
..607..493..3411..
..6074..93..3411..
9874327654
987437654
