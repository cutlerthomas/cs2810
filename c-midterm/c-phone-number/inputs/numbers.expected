(435) 555-1698
(435) 555-1698
Invalid input
(801) 123-9876
(801) 123-9876
Invalid input
Invalid input
(607) 493-3411
Invalid input
(987) 432-7654
Invalid input
